README.md

```diff
@@ -110,6 +110,8 @@ After this, Codacy displays the results of the analysis of your commits and pull
 
 To use the GitHub Action with Codacy integration:
 
+1.  On Codacy, [enable the containerized client-side tool](../../repositories-configure/configuring-code-patterns.md) and configure the corresponding code patterns on your repository **Code patterns** page.
```

Comment:
The reason I'm adding "On Codacy" is to clarify that these steps are performed on the Codacy UI instead of directly on GitHub, since users will be reading the instructions on the README of this repository.